src/storage/metrics.rs

```diff
@@ -5,6 +5,9 @@ use prometheus_static_metric::*;
 
 make_static_metric! {
     pub label_enum CommandKind {
+        get,
```

Comment:
Looks like this change is not necessary?

Reply:
This change will make metrics `tikv_storage_command_total` have type `get` , then `command` would have this type. I tested that without this change, `tikv_storage_command_total` doesn't have type `get`.

Reply:
yes, here I tested wrong before, and will remove it.